sviatoslav/src/main/java/com/cbu/edu/hw_01/ImitationStack.java

```diff
@@ -0,0 +1,89 @@
+package com.cbu.edu.hw_01;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * Created by Sviatoslav on 20.02.2017.
+ */
+public class ImitationStack<T extends Number> implements IGetIterator{
+    private int size;
+    private T[] stack;
+    private T[] newStack;
+    private int top;
+
+
+
+    public ImitationStack(int size) {
+        this.size = size;
+        stack = (T[])new Object[size];
+        this.top = -1;
+    }
+
+    public boolean isEmpty() {
```

Comment:
Note that you can simplify it by ternary operator - `return top == -1 ? true : false;`